docs/source/AdministratorGuide/Configuration/ConfReference/Resources/Sites/index.rst

```diff
@@ -47,7 +47,9 @@ This sub-subsection specify the attributes of each particular CE of the site. Mu
 +------------------------------------------------+-------------------------------------------------------------+--------------------------------+
 | *<CE_NAME>/architecture*                       | CE architecture                                             | architecture = x86_64          |
 +------------------------------------------------+-------------------------------------------------------------+--------------------------------+
-| *<CE_NAME>/CEType*                             | Type of CE, can take values as LCG or CREAM                 | CEType = LCG                   |
+| *<CE_NAME>/CEType*                             | Type of CE, can take values as LCG or CREAM                 | CEType = ARC                   |
```

Comment:
I thought we had agreed to get ride of that section of the doc, so is it still worth updating it ?

Reply:
You mean the whole ConfReference section?

Reply:
As long as there is no replacement for this part we should update the options, otherwise the options will be forgotten.